packages/e2e-tests/tests/playwright-01_basic-panel.test.ts

```diff
@@ -97,7 +98,7 @@ test("playwright-01: Basic Test Suites panel functionality", async ({
   const backButton = getTestRecordingBackButton(page);
   await backButton.click();
 
-  // Check if we're back on the main tests panel
+  debugPrint(page, "Check if we're back on the main tests panel");
   const secondRecordingTrees = getTestRecordingTrees(page);
   await waitFor(async () => {
```

Comment:
nit: I guess you felt like it added value to include these new logs? Their formatting is kind of inconsistently and might cause some confusion when mixed in with our other logs. 

For example, it means nothing to see "_can open tests_" in the console.

If you want to add these, that's fine, but let's update them to be more semantically meaningful?